src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -94,4 +95,22 @@ private function loadResources()
             unset($this->resourceFiles[$key]);
         }
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function warmUp($cacheDir)
+    {
+        if (null !== $this->options['cache_dir']) {
+            if (null !== $this->locale) {
+                $this->loadCatalogue($this->locale);
```

Comment:
hi @xavierleune  :D, just remove https://github.com/xavierleune/symfony/blob/feature/13919-translations-cache-warmer/src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php#L95 and it will work in dev env.